include/wx/qt/private/treeitemdelegate.h

```diff
@@ -67,6 +68,28 @@ class wxQTTreeItemDelegate : public QStyledItemDelegate
         QStyledItemDelegate::setModelData(editor, model, index);
     }
 
+    bool helpEvent(QHelpEvent *event, QAbstractItemView *view, const QStyleOptionViewItem &option, const QModelIndex &index)
+    {
+        if ( event->type() == QEvent::ToolTip)
+        {
+            const QRect itemRect = view->visualRect(index);
+            const QSize bestSize = sizeHint(option, index);
+            if ( itemRect.width() < bestSize.width() )
+            {
+                QString value = index.data(Qt::DisplayRole).toString();
+                QToolTip::showText(event->globalPos(), value, view);
```

Comment:
`event->globalPos()` is the position of the mouse cursor. Is this truly the correct place for the tooltip? Maybe a better place would be a "global position" calculated for the origin of `itemRect`, so that a tooltip would be shown over the item, and not start at mouse position. (I remember I've seen such tooltip in other list-like controls, don't remember where though.)

Reply:
If I get a chance, I'll implement this change.